### DISCUSSION TITLE
Hello from kohki-shikata: Supporting Live Sync Togehter

### DISCUSSION THREAD

**Discussion by kohki-shikata** [vrtmrz/obsidian-livesync#560] [General]
Hi there.

I'm kohki-shikata, vrtmrz assigned this repository as a new member.

I work as a full-stack engineer in Japan, also an Obsidian and Live Sync user. We, vrtmrz and I, met on the 17th of December and discussed the future of Live Sync and what I could support for it.

Currently we have decided, 

- Japanese localisation
- Create a document site
- Maintain issues and discussions

I will commit to these activities. And will comment on issues, discussions, reply to your messages.

Thank you for reading and for your continued support of Live Sync.

**Comment by vrtmrz** [comment]
I apologize for my late arrival, and I am also afraid I have been unwell again.

Today, I am so happy to report that I am back, and to introduce Kohki to you, which is a great pleasure and honour for me. He is a valued friend of ours who will make many contributions.

He lives close to me and I believe this will also help with this and the viability of the community should something serious happen to me. I have also met him face to face and I am sure he shares my values and will be a valuable asset to our community.

Just to be clear, this is not to deny diversity or other possibilities. We believe that working together with you is, of course, also important. However, due to the nature of the project, I would like to meet with you to discuss key decisions. I would always welcome a "We need to talk"(If it means also the same in the U.K. or not) person! (although my English is not very good. I am counting on your kindness, thank you).
